### ISSUE TITLE
Unify Mapping results

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/Goliath#11]
See also #4 for the long discussion on this topic.

I would prefer to use CHEBI:1234 ; simply because then people see where the identifier comes from with more ease. HMDB also includes their database name in the identifier, and Ensembl does this too (even with a code to see which species the ID belongs too). And Pathvisio can handle both ChEBIs (so CHEBI:1234 and 1234). About the secondary IDs: it would be great if we could do such a mapping at some point (as well as for outdated IDs from other databases), however these are very specific use cases, so I wouldn't bother with it now. Therefore, I believe the correct format for metabolite IDs should be:

- [ ] ChEBI   CHEBI:1234
- [ ] HMDB HMDB0012345 (so with a length of 11, not 9, that was the old structure of the ID).
- [ ] Inchi+InchiKey Not sure about these, what do you think @egonw ... At Wikidata the Inchi is not given with the prefix "InChI=", at wikipedia this is given (because they add the InchiKey under the same database (and given a prefix "Key"); we do not do that in BridgeDb right?).
- [ ] please add other specific examples if needed.

**Comment by egonw** [comment]
Yes, "InChI=1S/" is integral part of the InChI and must be included. BTW, the "1S" is version info and can be different.

I'm still hesitant calling an InChIKey an identifier, but if that is what we want. But then we must make a second DataSource.
